test/java/scenarios.json

```diff
@@ -40,5 +40,17 @@
                 "installMaven": true
             }
         }
+    },
+    "install_gradle_and_maven_with_specific_version": {
```

Comment:
Looks like `install_gradle_and_maven_with_specific_version.sh` test file is missing in the PR?